pkg/universe.dagger.io/x/berryphillips@gmail.com/onepassword/example/plan.cue

```diff
@@ -0,0 +1,41 @@
+package example
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"universe.dagger.io/x/berryphillips@gmail.com/onepassword"
+)
+
+dagger.#Plan & {
+	client: env: {
+		OP_ADDRESS:    dagger.#Secret
+		OP_EMAIL:      dagger.#Secret
+		OP_SECRET_KEY: dagger.#Secret
+		OP_PASSWORD:   dagger.#Secret
+	}
+
+	actions: {
+		actionsToken: onepassword.#Read & {
+			credentials: onepassword.#Credentials & {
+				address:   client.env.OP_ADDRESS
+				email:     client.env.OP_EMAIL
+				secretKey: client.env.OP_SECRET_KEY
+				password:  client.env.OP_PASSWORD
+			}
+			reference: onepassword.#Reference & {
+				vault: "MyVault"
+				item:  "GitHub"
+				field: "actions-token"
+			}
+		}
+		_alpine: core.#Pull & {source: "alpine:3"}
+		example: core.#Exec & {
+			input: _alpine.output
+			args: [
+				"echo",
+				actionsToken.output,
```

Comment:
If you use `core.#NewSecret`, this will need to change to either 
1. load the secret as a file: https://github.com/dagger/dagger/blob/bd948c41ecb074c9460d138791c25140625c578f/tests/tasks/newsecret/newsecret.cue#L32-L35
2. or as and ENV: https://github.com/dagger/dagger/blob/d4548195bb37dd07b8cdcc5b54021adf01c0814b/pkg/dagger.io/dagger/core/exec.cue#L22

```cue
		example: core.#Exec & {
			input: _alpine.output
      env: SECRET: actionsToken.output
			args: [
				"echo",
				"$SECRET",
			]
			always: true
		}

```

Reply:
Thanks for providing examples and links. Really helpful!!